edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Microsoft.Azure.Devices.Edge.Agent.Core.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\dotnetVersion.props" />
+
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <Configurations>Debug;Release;CheckInBuild</Configurations>
     <HighEntropyVA>true</HighEntropyVA>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Akka.Streams" Version="1.3.13" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.3" />
```

Comment:
Please use Visual Studio and right click on solution in solution explorer, select "Manage NuGet Packages for solution" and check "Consolidate" tab to make sure no multiple version of any package.  thanks.

Reply:
I checked and updated, but there is one remaining, in EdgeDownstreamDevice project, it references Microsoft.Azure.Devices.Client of version 1.20.3-asc-edge, but others use 1.23.1.  I think we should use 1.23.1 for that project as well, please confirm with @varunpuranik, thanks.

Reply:
I also updated some nuget package as well.

Reply:
Yes I think we can change. This project is used for tutorials. We should keep the SDK versioning the same as our product.